### DISCUSSION TITLE
deselection

### DISCUSSION THREAD

**Discussion by kujohn** [helix-editor/helix#5133] [Q&A] [answered]
Is there a way to deselect? Use case would be navigating by word but instead of modifying the entire word, I would just like to append to the word or select the first letter to modify, Currently, it involves moving the selection away and back with`h, l`.

**Comment by gabydd** [comment, marked as answer]
Collapsing a selection can be done with ``;``. I would also suggest doing the tutor with ``hx --tutor`` or ``:tutor`` in the editor. You can also check out the key map docs [https://docs.helix-editor.com/](https://docs.helix-editor.com/) and the command pallete(``<space>?``) for quick reference.

**Comment by markand** [comment]
I've stambled accross this issue while reading the tutor. Sometimes, pressing `f` and next a character that may appear only at the end of file would select the entire file. But `;` collapse selection at the end of selection. Do I miss something or I can not find a keymap to just cancel the selection and get back to the original location?

**Reply by waddie** [reply]
You can flip which end of selections the cursors are on with `A-;`. Do that before collapsing the selection.

**Reply by markand** [reply]
Thanks, will map a keybind to it to make it easier.
